app/models/concerns/form526_claim_fast_tracking_concern.rb

```diff
@@ -165,7 +165,79 @@ def get_claim_type
     end
   end
 
+  # Contact the VRO classifier service to classify the contentions on this form
+  # update the form with the classification codes
+  # returns true if all of form's contentions were classified
   def update_classification!
+    if Flipper.enabled?(:disability_526_classifier_multi_contention)
+      update_contention_classification_all!
+    else
+      update_contention_classification_single_contention!
+    end
+  end
+
+  def update_form_with_classification_codes(classified_contentions)
+    classified_contentions.each_with_index do |classified_contention, index|
+      classification_code = classified_contention['classification_code']
+      if classified_contentions[index]['classification_code'].present?
+        form[Form526Submission::FORM_526]['form526']['disabilities'][index]['classificationCode'] = classification_code
+      end
+    end
+
+    update!(form_json: form.to_json)
+    invalidate_form_hash
+  end
+
+  def classify_vagov_contentions(params)
+    vro_client = VirtualRegionalOffice::Client.new
+    response = vro_client.classify_vagov_contentions(params)
+    response.body
+  end
+
+  def format_contention_for_vro(disability)
+    contention = {
+      contention_text: disability['name'],
+      contention_type: disability['disabilityActionType']
+    }
+    contention['diagnostic_code'] = disability['diagnosticCode'] if disability['diagnosticCode']
```

Comment:
Might want to do `if disability['diagnosticCode'].present?` because the empty string (not sure if that's ever possible for this field) evaluates to truthy.